template/base/_pnpm-workspace.yaml

```diff
@@ -1,4 +1,9 @@
 # managed by skuba
+minimumReleaseAge: 4320
```

Comment:
Not really fussed, but thoughts on:

```suggestion
minimumReleaseAge: 4320 # 3 days
```

Reply:
Like it saves people having to ask what it means 😆